app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/JourneyBaseController.scala

```diff
@@ -50,7 +49,7 @@ abstract class JourneyBaseController[Journey](
       )
     }
 
-  final def actionReadJourney(body: Request[_] => Journey => Future[Result]): Action[AnyContent] =
+  final def actionReadJourney(body: RequestWithSessionData[_] => Journey => Future[Result]): Action[AnyContent] =
```

Comment:
@steve-edmans why do you need `RequestWithSessionData` ?

Reply:
@arturopala we need an implicit instant of Messages for the twirl templates. This is included with the RequestWithSessionData